website/src/views/hooks/useMediaQuery.test.tsx

```diff
@@ -0,0 +1,36 @@
+import { act, render } from '@testing-library/react';
+import { breakpointUp } from 'utils/css';
+import { FC } from 'react';
+import { mockWindowMatchMedia } from 'test-utils/mockDom';
+import useMediaQuery from './useMediaQuery';
+
+const Tester: FC = () => {
+  const matchedBreakpoint = useMediaQuery(breakpointUp('md'));
+
+  if (matchedBreakpoint) {
+    return <div />;
+  }
+  return null;
```

Comment:
Hmm I think this tester component can be more explicit, which will probably make the test case `expect`s clearer

```suggestion
  return matchedBreakpoint ? <>matched</> : <>no match</>);
```

Reply:
Oh btw, I think it'll be easiest to use inline snapshots with this, e.g.

```
expect(container).toMatchInlineSnapshot()
```

I think the snapshot will be something like `<div>matched</div>`, which will be extremely clear with minimal effort on your part.